### ISSUE TITLE
Setting trackers: [] in the config file does not search all trackers

### ISSUE THREAD

**Issue by newadventure079** [mmgoodnow/cross-seed#155]
In the config file, it says
```
// Trackers to search
// Set to [] if you want to search all trackers.
// Tracker ids can be found in their Torznab feed paths
trackers: [], 
```

So I did that and in the output below, it shows `trackers: '',` where I'd expect it to list all the trackers or even maybe say `All`
```
cross-seed search --verbose --torrent-dir /volume1/data/Torrents/cross-seed/cross-seed-input/tmp/
verbose: [configdump] {
  jackettServerUrl: 'http://172.17.0.1:9117',
  jackettApiKey: '[REDACTED]',
  trackers: '',
  torrentDir: '/volume1/data/Torrents/cross-seed/cross-seed-input/tmp/',
  outputDir: '/volume1/data/Torrents/cross-seed/cross-seed-output',
  searchAll: false,
  verbose: true,
  action: 'save',
  rtorrentRpcUrl: 'http://172.17.0.1:10000/RPC2',
  offset: 0,
  delay: 15,
  includeEpisodes: false
}
```

And it really isn't searching any trackers because in the Jackett log file, there is:
```
2021-12-01 12:01:13.8958 Info Manual search in  for 13 Reasons Why S02 => Found 0 releases
```
Note the space between `in  for`, where the tracker names would be outputted

But when I override `trackers: []` and input some trackers, it actually does search the trackers and finds releases
```
2021-12-01 12:02:36.6832 Info Manual search in xxx,xxx,xxx,xxx,xxx,xxx for 13 Reasons Why S02 => Found 183 releases
```

**Comment by Jerrk** [comment]
I believe i have the same issue

`Info 	Manual search in  for Tenacious D in The Pick of Destiny 2006 => Found 0 releases`

Running in docker on unraid if that matters

**Comment by mmgoodnow** [comment]
fixed in v2.9.2, thanks for the sleuthing!
